Update and rename 2021-03-03-gesichter-und-nummernschilder-auf-bilder… (#31)
mmmint-ai/mmmint.ai: drafts/2021-03-05-gesichter-und-nummernschilder-auf-bildern-anonymisieren.md

 {% include image_compare_block.html section=page.compare.city %}

> Sowohl die Person zwischen den Autos, obwohl Sie nicht direkt in die Kamera blickt, als auch das Nummernschild des roten SUVs werden von unserer Künstlichen Intelligenz automatisiert und erkannt und anonymisiert. Das Bild verliert dabei nicht an Aussagekraft und die sensiblen Daten werden entfernt. Datenschutz sollte Sie nicht daran hindern Ihr Bildmaterial für Marketingzwecke effizient zu verwenden - nutzen Sie unseren Service, um in Echtzeit Ergebnisse erzielen zu können.

Reviewer: ```suggestion
Sowohl die Person zwischen den Autos, obwohl Sie nicht direkt in die Kamera blickt, als auch das Nummernschild des roten SUVs werden von unserer Künstlichen Intelligenz erkannt und automatisiert anonymisiert. Das Bild verliert dabei nicht an Aussagekraft und nur die personenbezogenen Daten wurden entfernt. Datenschutz sollte Sie nicht daran hindern Ihr Bildmaterial für Marketingzwecke zu verwenden - nutzen Sie unseren Service, um schnelle Ergebnisse erzielen zu können.
```

## Wie kann ich den Anonymizer-Service nutzen?

Die Einbinung des Anonymizer-Service in Ihre Systeme ist dank einer standardisierten Schnitstelle einfach und schnell möglich. Aktuell wird der [<mark>Anonymizer-Service</mark>](/solutions/anonymizer/) als REST-API zur Verfügung gestellt und bietet Entwicklern damit einen einfachen Zugang zu unserer Künstlichen Intelligenz. [Hier finden Sie unsere Dokumentation](/solutions/anonymizer_api.html)

Das Frontend für den Upload von Bildern & Videomaterial befindet sich aktuell in der Entwicklung. Sie möchten mit Ihrem Anwendungsfall einen Pilot mit uns starten? Sprechen Sie uns gerne an! Wir freuen uns auf einen Austausch und sind gespannt auf Ihren Anwendungsfall.

> > "Hier sehen Sie einen Auszug aus der Plattform Ebay Kleinanzeigen. Wir sind uns sicher, das geht auch leichter."

Reviewer: ```suggestion
> "Die Zeiten der lästigen, manuellen Anonymisierung von Bildern ist nun vorbei. Wir sind uns sicher, das geht auch leichter."
```

Ich weiß nicht ob der Screenshot von E-Bay ist, hast du eine Quelle? 
Habe dazu auch leider nur den unten stehenden Link gefunden.

![Opel Zafira manuelle Anoymisierung von Auto Kennzeichen](/images/solutions/anonymizer/opel_zafira_crop.jpeg)

<section id="call-to-action" class="block cta-block bg-accent outer">
  <div class="inner-large">
    <div class="grid">
      <div class="cell block-content">

>         <h2 class="block-title">Jetzt Zeit sparen...</h2>

Reviewer: ```suggestion
        <h2 class="block-title">Jetzt T-Shirt wieder anziehen...</h2>
```

Reviewer: @maxleimkuehler and I have been in a call and prefer the T-Shirt option. @maximilianstein
